steps/idam/IdamLogin/IdamLogin.js

```diff
@@ -0,0 +1,28 @@
+const { Question } = require('@hmcts/one-per-page');
+const { redirectTo } = require('@hmcts/one-per-page/src/flow');
+const { form, text } = require('@hmcts/one-per-page/forms');
+const Joi = require('joi');
+const paths = require('paths');
+
+class IdamLogin extends Question {
+  static get path() {
+    return paths.idam.mockIdamLogin;
```

Comment:
shoudn't this path be the real idam login path ?

Reply:
Looks like all instances used as a mock 
const IdamMockLogin = require('steps/idam/IdamLogin/IdamLogin');

I agree on this one , unless it's temporary solution the path of the file is not under mock so it's little confusing.

Reply:
Done

Reply:
@iguannago no, not the IDAM path, this is the path of the step that uses the idam middleware. The middleware redirects to the actual service. However, it should not include `mock` in the name of the step so I have rectified that.